extension/src/popup/styles/global.scss

```diff
@@ -35,6 +35,7 @@ a {
 
 #root {
   position: relative;
+  height: 100%;
```

Comment:
the PopupWrapper needed this to be full height (so can center content correctly). Looks like it lost it when we removed [this wrapper](https://github.com/stellar/freighter/pull/405/files#diff-d9ab29cfb513dab891d9a6d041b202d627e8895ec8d8dbaad90514e1eab61f5dL97-L99)

Reply:
Good catch! I removed this, looked around, and things *appeared* to be fine. Classic 🤦